docs/pages/preview/eas-update/getting-started.md

```diff
@@ -0,0 +1,164 @@
+
+   Whenever we change native code, we'll need to iterate the runtime version and create new builds. This way, and updates we publish will also target the new state of the build's native code.
+
+4. To set up the configuration file for builds, run:
+
+   ```bash
+   eas build:configure
+   ```
+
+   Then follow the prompts.
+
+   This will create a file named **eas.json**. Inside the `release` profile, add the `channel` property with a value of `"production"`:
+
+   ```bash
+   {
+     "build": {
+        "release": {
+          "channel": "production"
+        },
+       ...
+     }
+   }
+   ```
+
+   This `channel` property will allow you to point updates at builds with this channel. Later, if you set up a GitHub Action to publish changes on merge, it will make it so we can merge code into the `production` branch, then those commits will be published and made available to builds with the channel `"production"`.
+
+5. Finally, we need to create a `channel` and `branch` named `production` on EAS' servers. We can accomplish the creation of these with this command:
+
+   ```xml
+   eas channel:create production
+   ```
+
+## Configuring expo-asset and expo-updates
+
+> ⚠️ These steps are only required temporarily. These changes will be included in SDK 43 (end of Sept 2021).
+
+To successfully load updates, we'll need the latest versions of the `expo-asset` and `expo-updates` packages from expo/expo. You can download these libraries below and drag them into the parent folder of your project:
+
+[expo-asset-8.3.3.tgz](https://drive.google.com/file/d/1nXEO62-sVz_LRSyitW9EK3Chg8w7TUat)
+
+[expo-updates-0.8.4.tgz](https://drive.google.com/file/d/170WIMgFE7tzL362xFd3r9pfmSQNW6M2c)
+
+When you download these files, they will have the **.tar** extension. Change it back to **.tgz**.
+
+Once those two files are in your project, run:
+
+```json
+npm install *.tgz
+```
+
+After running this command, inside **package.json**, you should see:
+
+```json
+"expo-asset": "file:expo-asset-8.3.3.tgz",
+...
+"expo-updates": "file:expo-updates-0.8.4.tgz",
+```
+
+- Advanced: build these libraries locally. You can build the packages yourself by checking out `sdk-42` in expo expo, and running
+
+  ```json
+  npm pack ~/your-path/expo/packages/expo-updates .
+  npm pack ~/your-path/expo/packages/expo-asset .
+  ```
+
+## Creating a build for the project
+
+Next, we'll need to create iOS and/or Android build. [Learn more](/build/setup).
+
+## Publishing an update
+
+Now we're ready to publish an update to the builds created in the last section.
+
+1. Make any desired changes to your project.
+2. Then publish an update with the following command:
+
+   ```xml
+   eas branch:publish --auto
```

Comment:
if we are going to use `--auto` here we should not direct the user to make the `channel = production`, but instead `channel = <current Git branch>`